src/backend/web/validation.js

```diff
@@ -51,7 +51,7 @@ function validateNewFeed() {
   };
 }
 
-const validateParam = (req, res, next) => {
+const validatePostsIdParam = (req, res, next) => {
   param('id').isLength({ min: 10, max: 10 })(req, res, (err) => {
```

Comment:
Hi, One little twist. Can you follow the same pattern as in the following commit 
https://github.com/Seneca-CDOT/telescope/pull/1339/commits/803fb2e64aecbc39459910907ed3f071f1c23b26
This one the person is doing the same thing as you do. See what I mean by return a middleware. Then what you left is the unit tests.

Reply:
Hey, @tonyvugithub  So Need to change his format like this?

const validatePostsIdParamRules = [
  param('id','Id Length is invalid').isLength({ min: 10, max: 10 })
]; 

const validatePostsIdParam = () => {
  return async (req, res, next) => {
    await Promise.all(validatePostsIdParamRules.map((rule)=>rule.run(req)));

    const result = validationResult(req);
    if(result.isEmpty()){
      return next();
    }
    const errors = result.array();
    return res.status(400).send(errors);
    
  };
};

I changed it with his style. Is it right?

Thanks :)